agenta-backend/agenta_backend/tests/unit/test_evaluators.py

```diff
@@ -344,130 +352,3442 @@ def test_auto_levenshtein_distance(output, data_point, settings_values, expected
         "message": "aloha",
     },
     "trace": {
```

Comment:
@aybruhm @jp-agenta 
Can we find a better approach to parameterize the trace please.. it will be hard to work on 3k line file..